wemake_python_styleguide/visitors/ast/conditions.py

```diff
@@ -1,11 +1,12 @@
 import ast
 from collections import defaultdict
 from functools import reduce
-from typing import ClassVar, DefaultDict, List, Mapping, Optional, Set, Type
+from typing import ClassVar, DefaultDict, Final, List, Mapping, Set, Type
```

Comment:
We import `Final` from `typing_extensions`, because we support `python3.6` and `python3.7`